### PR TITLE
Fix daily run

### PR DESCRIPTION
**Pull request by Sgitario**
- Upgrade Quarkus Test Framework to 0.0.3
- Detect modules to be executed as part of the Native CI run. Fix https://github.com/quarkus-qe/quarkus-test-suite/issues/60
- Delete mention to ephemeral namespaces. Fix https://github.com/quarkus-qe/quarkus-test-suite/issues/65
- Fix sql-app modules by not using shared properties (this change is derived by the latest version of the test framework) and using integers instead of longs.
- Update Quarkus CLI and Continuous Testing latest changes
- When creating applications using the Quarkus CLI, we can't start on DEV mode. This will be fixed when using Quarkus Test Framework 0.0.4.
- Bump versions of: quarkus-qpid-jms, formatter-maven-plugin, impsort-maven-plugin and xml-format-maven-plugin.

### CODE REVIEW

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java
Comment (Sgitario) [inline quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java:74]: I'm relaxing this assertion as for 999-SNAPSHOT, it's returning 2.0.2.Final dependencies which is correct, but I don't want to hardcode 2.0.2.Final here.

**Comment by Sgitario** [comment]
@rsvoboda I think we're good to review and merge this one.

File: http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java
Comment (rsvoboda) [inline http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java:28]: Commits https://github.com/quarkus-qe/quarkus-test-suite/pull/72/commits/4911e0536e736952e4707bacbfe3a9513ed90966 and https://github.com/quarkus-qe/quarkus-test-suite/pull/72/commits/e782a626bc4c7bfa00083567583351598b95944f are reverts to each other, the only relevant part is `@Disabled("TODO: Fix flaky test: https://github.com/quarkus-qe/quarkus-test-framework/issues/146")`

Please squash them.

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java
Comment (Sgitario) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java:31]: I don't think we need to maintain and hence parametrise this. This is only to verify we can fetch other versions using the CLI.

File: sql-db/sql-app/src/test/java/io/quarkus/ts/openshift/sqldb/AbstractSqlDatabaseIT.java
Comment (Sgitario) [reply sql-db/sql-app/src/test/java/io/quarkus/ts/openshift/sqldb/AbstractSqlDatabaseIT.java:30]: This is caused by a change in the hamcrest dependency whereby `is(8L)` was working fine, but not anylonger. I had to change it to `is(8)`. Actually, this is how it is in the rest of test suite.

File: http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java
Comment (Sgitario) [reply http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java:28]: Thanks for spotting! Fixed now.

File: sql-db/sql-app/src/test/java/io/quarkus/ts/openshift/sqldb/AbstractSqlDatabaseIT.java
Comment (rsvoboda) [reply sql-db/sql-app/src/test/java/io/quarkus/ts/openshift/sqldb/AbstractSqlDatabaseIT.java:30]: That's interesting, thanks for explanation

**Review by rsvoboda — APPROVED** [review]
Approving, let's wait for the last CI job before merging

**Pull request merged**
